docs/reference/api/queues/send-tasks.mdx

```diff
@@ -39,6 +39,38 @@ Using one of the provided Nitric SDKs for your language is the easiest way to wo
 
 Every task that's produced can be uniquely identified so that receivers can be idempotent. To assist with this, Nitric Tasks include an ID property which should contain the unique ID for that task. By default, the Nitric Membrane will automatically generate this ID as a v4 UUID for every published task if it doesn't already contain an ID. Custom ID creation should be implemented by your code to ensure that duplicate queue items contain the same ID value.
 
+<CodeExamples
+	languages={[
+	{
+		label: "Node",
+		value: "node"
+	},
+	{
+		label: "Python",
+		value: "python"
+	},
+	{
+		label: "Java",
+		value: "java"
+	},
+	{
+		label: "Go",
+		value: "go"
+	}
+	]}
+	defaultLang="node"
+>
+
+<CodeSnippet snippet="nitrictech/node-sdk/examples/queues/send-id.ts" />
```

Comment:
incorrect according to: https://github.com/nitrictech/node-sdk/pull/77/files

Reply:
updated filename